packages/components/psammead-media-player/src/index.stories.jsx

```diff
@@ -18,23 +18,20 @@ storiesOf('Components|MediaPlayer', module).add('Default', () => (
   />
 ));
 
-storiesOf('Components|Media Player', module).add(
-  'Default with guidance',
-  () => (
-    <CanonicalMediaPlayer
-      src="https://www.test.bbc.co.uk/ws/av-embeds/articles/c3wmq4d1y3wo/p01k6msp/en"
-      placeholderSrc="https://ichef.bbci.co.uk/news/640/cpsdevpb/4eb7/test/ba7482d0-cca8-11e8-b0bf-f33155223fc4.jpg"
-      service="news"
-      mediaInfo={{
-        title: 'Dog chases cat.',
-        guidance: 'Contains strong language with adult humor',
-        ...withDuration,
-      }}
-    />
-  ),
-);
+storiesOf('Components|MediaPlayer', module).add('Guidance', () => (
+  <CanonicalMediaPlayer
```

Comment:
The Media Player itself requires a `title` attribute. Please refer to the other stories.
This will fix the console error.